src/Tizen.NUI/src/public/InputMethodContext.cs

```diff
@@ -40,6 +40,9 @@ public class InputMethodContext : BaseHandle
         /// Constructor.<br/>
         /// </summary>
         /// <since_tizen> 5 </since_tizen>
+        /// This will be deprecated
+        [Obsolete("Deprecated in API8; Will be removed in API11")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
The first comment seems unnecessary. 
Our current deprecation policy states that API will be deprecated in N version and removed in N+2 version. Therefore it should ```Obsolete("Deprecated in API8; Will be removed in API10")```
Please don't use EditorBrowsable attribute while using the Obsolete attribute. This API should remain public until its removal.

Reply:
Done